### DISCUSSION TITLE
Return information about contract

### DISCUSSION THREAD

**Discussion by alex-pakalniskis** [graphprotocol/graph-tooling#1826] [Ideas]
## User request
Add a Graph CLI command to return information about a smart contract in human-readable format. Include data such as:
* start block
* ABI, if available
* source code, if available
* all chains where this contract is deployed

By default, return data in terminal but enable users to save data to file. 

## Example commands

``` bash
graph contract <0x...> <network> # returns start block, ABI, & source code
graph contract <0x...> <network> --save-abi abiFile.json # save ABI to file
graph contract <0x...> <network> --save-code Contract.sol # save source code to file
graph contract <0x...> <network> --save-chains chains.json # save list of chains where contract is deployed
graph contract <0x...> <network> --save # save ABI, source code, and chains list to file
```

**Comment by matthewdarwin** [comment]
Duplicate of https://github.com/graphprotocol/graph-tooling/issues/1827
